adcm_client/objects.py

```diff
@@ -301,6 +302,7 @@ def update(d, u):
             return d
         # this check is incomplete, cases of presence of keys "config" and "attr" in config
         # are not considered
+        allure_attach_json(data, name="Changed fields")
         is_full = "config" in data and "attr" in data
         config = self.config(full=is_full)
```

Comment:
```suggestion
        config = self.config(full=is_full)
        allure_attach_json(config, name="Original config")
```

Reply:
original config will be attached in `config_set` function

Reply:
fix